README.md

```diff
@@ -1,124 +1,88 @@
-- Support local use of Camel debugger by attaching to a running Camel route written in Java, Yaml or XML (only `Camel Main` mode for XML) using the PID
+<h1 align="center">
+ <img width="125" height="125" src="https://raw.githubusercontent.com/camel-tooling/camel-dap-client-vscode/main/icons/icon128.png">
+ <br>
+ Debug Adapter for Apache Camel
+</h1>
+
+<p align="center">
+ <a href="https://marketplace.visualstudio.com/items?itemName=redhat.vscode-debug-adapter-apache-camel"><img src="https://img.shields.io/visual-studio-marketplace/v/redhat.vscode-debug-adapter-apache-camel?style=for-the-badge" alt="Marketplace Version"/></a>
+ <a href="https://github.com/camel-tooling/camel-dap-client-vscode/actions/workflows/ci.yaml"><img src="https://img.shields.io/github/actions/workflow/status/camel-tooling/camel-dap-client-vscode/ci.yaml?style=for-the-badge" alt="Main CI"></a>
+ <a href="https://github.com/camel-tooling/camel-dap-client-vscode/blob/main/LICENSE.txt"><img src="https://img.shields.io/github/license/camel-tooling/camel-dap-client-vscode?color=blue&style=for-the-badge" alt="License"/></a>
+ <a href="https://gitter.im/camel-tooling/Lobby"><img src="https://img.shields.io/gitter/room/camel-tooling/Lobby?color=yellow&style=for-the-badge" alt="Gitter"/></a>
+</p><br/>
+
+<h2 align="center">DAP Client for Visual Studio Code.</h2>
+
+<p align="center">
+  <a href="#features">Features</a> •
+  <a href="#requirements">Requirements</a> •
+  <a href="#how-to-debug">Example</a> •
+  <a href="https://camel-tooling.github.io/camel-dap-client-vscode/">Documentation</a> •
+  <a href="#issues">Issues</a>  •
+  <a href="#data-and-telemetry">Telemetry</a>
+</p>
+
+<p align="center">
+This is the <a href="https://code.visualstudio.com/">Visual Studio Code</a> extension that adds <a href="https://camel.apache.org/manual/debugger.html">Camel Debugger</a> power by attaching to a running Camel route written in Java, Yaml or XML DSL.
+</p><br/>
+
+<p align="center"><img src="./docs/images/breakpoint.png" alt="A breakpoint hit on a Camel route endpoint and the variables displayed." width="100%"/></p>
+
+### Features
+
+- `Camel Main` mode for XML only (It implies that it is not working with Camel context specified in Camel XML file.)
+  - Support use of Camel debugger by attaching to a running Camel route written in Java, Yaml or XML using the JMX Url
+  - Support local use of Camel debugger by attaching to a running Camel route written in Java, Yaml or XML using the PID
 - Support a single Camel context
 - Add/Remove breakpoint
-- Support conditional breakpoint with `simple` language. See [here](https://camel.apache.org/components/latest/languages/simple-language.html) for details on how to write condition with simple language.
+- Support conditional breakpoint with `simple` language. See [How to write condition with simple language](https://camel.apache.org/components/latest/languages/simple-language.html) for details.
```

Comment:
WDYT?
Support conditional breakpoint with simple language. -> Support conditional breakpoint**s** with simple language.

Not some specific breakpoint but some variety of it